scripts/makebumpver

```diff
@@ -28,14 +28,17 @@ import argparse
 import datetime
 import logging
 
+from jira import JIRA, JIRAError
```

Comment:
Please document somewhere how to get this module. I don't think I have it on my machine...

Reply:
Seems we are getting a bit too wild on improvements for RHEL branch... but why not.

Reply:
Thanks! This is great. I was expecting just something like `from jira import JIRA  # dnf install python3-jira`